src/main/java/fi/metatavu/dcfb/server/items/ItemController.java

```diff
@@ -29,6 +32,8 @@
 
 @ApplicationScoped
 public class ItemController {
+  
+  private static long RESERVATION_EXPIRE_MINUTES = 5l;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this field "RESERVATION_EXPIRE_MINUTES" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. ([squid:S3008](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS3008))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=18&issues=AWVh4kVHUj60SLv4Ylyn&open=AWVh4kVHUj60SLv4Ylyn)

[ISSUE_KEY]: AWVh4kVHUj60SLv4Ylyn